vortex-array2/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vortex-array2"
```

Comment:
I wonder if there's a way to get intellij to generate you the other format. Anyway we should unify these across all files on whatever is easier to get right